homeassistant/components/matter/config_flow.py

```diff
@@ -34,6 +34,7 @@
 ADDON_SETUP_TIMEOUT = 5
 ADDON_SETUP_TIMEOUT_ROUNDS = 40
 DEFAULT_URL = "ws://localhost:5580/ws"
+DEFAULT_TITLE = ""  # empty string = same name as integration
```

Comment:
It is a name already, so it isn't translatable. So, just "Matter" should work just fine.

```suggestion
DEFAULT_TITLE = "Matter"
```